source/kraken/realtime.cpp

```diff
@@ -281,6 +247,69 @@ static bool is_added_stop_time(const transit_realtime::TripUpdate& trip_update)
     return false;
 }
 
+static bool is_handleable(const transit_realtime::TripUpdate& trip_update,
+                          nt::PT_Data& pt_data,
```

Comment:
This should probably be `const` (maybe a const access has to be added, but it'd be safer)

Reply:
:+1: